src/Symfony/Component/Console/Command/Command.php

```diff
@@ -569,6 +569,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return Command The current instance
```

Comment:
In this class `@return Command` is used, should change this to `return $this` in the whole class ?